tests/plugin_tests/test_chan_track.py

```diff
@@ -109,3 +119,67 @@ def test_channel_members():
     assert 'foo1' in chan.users
     on_quit('foo1', conn)
     assert 'foo1' not in chan.users
+
+
+NAMES_FLOW = [
```

Comment:
```suggestion
TEST_COMMAND_SET = [
```

`NAMES_FLOW` is somewhat ambiguous here